src/util.ts

```diff
@@ -452,7 +462,97 @@ export function createMultiLinkFromLinks(links: string[]): string {
 		hashString += `&${key}=${value}`
 	}
 
-	return url.toString()
+	const shortenedLink = shortenMultilink(url.toString())
```

Comment:
If I understand correctly, it currently breaks multilinks on different chains.

Say we have a multilink `...?c=10,324&i=78,77...`. Normally it would mean deposit 78 on chain 10 + deposit 77 on chain 324. But now:
1. It will be automatically converted to `...?c=10,324&i=(77,1)...` and the regex in `isShortenedLink` will fail since `c=` is in an unexpected format
2. Even if we properly expand the link, it will become `...?c=10,324&i=77,78...` => deposit 77 will be looked up on chain 10 & deposit 78 will be looked up on chain 324, which is not correct

Reply:
I've been thinking and am not sure how we can do this best, lets sync on this (maybe together with hugo), and also see if its urgent to do rn, cause adds quite some complexity imo.